src/include/buffer.h

```diff
@@ -729,7 +729,9 @@ inline namespace v14_2_0 {
 
       void advance(int o) = delete;
       void advance(unsigned o);
+#if !defined(__i386__) && !defined(__arm__)
```

Comment:
i'd prefer the change at https://github.com/ceph/ceph/pull/25729/commits/1309f1553eced132392a307073c07738deab81f1#diff-e2d9d51a484aa6d89093b0bcfc70a1e0. lemme send a PR to address this.

Reply:
Agreed - that is neater

Reply:
#28278 is posted to address this issue.